### PR TITLE
Make lcli docker image portable 

### LINKED ISSUE #4370: sigp/lcli container image only built for modern x86 CPUs
## Description

The lcli binary shipped in the [sigp/lcli container image](https://hub.docker.com/r/sigp/lcli/tags) is built to run on a CPU with extensions (probably AVX2?) that are not present on all x86 CPUs. The result is that when the tool is run on such a machine, it throws an illegal instruction exception.

Such CPUs are not particularly rare : we have encountered the issue both in GitHub runners and in VMs in our internal build infra.

## Version

Tested [this container image](https://hub.docker.com/layers/sigp/lcli/latest/images/sha256-37097766174d577210244d0f3164a8878ccee8e1eac5866eea3e22471568fb3c?context=explore) and also a local built from the current stable branch.

## Present Behaviour

The `lcli` tool, when run with the `insecure-validators` subcommand, throws an illegal instruction exception on some CPUs.

e.g.:

```
$  ./lcli insecure-validators --count 4 --base-dir $(pwd) --node-count 4
Validator 1
Illegal instruction (core dumped)
```

## Expected Behaviour

Should not throw illegal instruction exception.

## Steps to resolve

The lcli container should be built with a portable compiler target, but it isn't.

### PR DESCRIPTION
**Pull request by aBMania**
## Issue Addressed

Fixes #4370 

## Proposed Changes

Use `FEATURES=portable` instead of `PORTABLE=true` as proposed [here](https://github.com/sigp/lighthouse/issues/4370#issuecomment-1579635525)

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -152,10 +152,12 @@ jobs:
...
-                  docker push ${LCLI_IMAGE_NAME}:${VERSION}${VERSION_SUFFIX}
+            - name: Build lcli and push
+              uses: docker/build-push-action@v5
+              with:
+                  build-args: |
+                      FEATURES=portable
+                  context: .
+                  push: true
+                  file: ./lcli/Dockerfile
+                  tags: ${{ env.LCLI_IMAGE_NAME }}:${{ env.VERSION }}${{ env.VERSION_SUFFIX }}
```
Comment (dapplion) [inline .github/workflows/docker.yml:163]: This syntax is valid for `docker/build-push-action` and is equivalent to the previous manual command
https://github.com/docker/build-push-action/blob/94d76d3bc1409736cb5dc1ada9502bec3a72973c/__tests__/context.test.ts#L89

**Review by dapplion — APPROVED** [review]
Built and run the updated dockerfile + CI command locally, on an x86 machine tho

**Comment by michaelsproul** [comment]
@mergifyio queue

**Comment by michaelsproul** [comment]
Pushed a new commit to re-run the target branch check

**Comment by michaelsproul** [comment]
@Mergifyio refresh

**Comment by michaelsproul** [comment]
@Mergifyio requeue

**Comment by michaelsproul** [comment]
@Mergifyio queue

**Pull request merged**